dist/clouduploader.rb

```diff
@@ -73,12 +73,13 @@ def upload_image_to_ec2(image, credentials, region, filename)
 end
 
 if platform == 'ec2'
-  region = data['region']
+
   arn = data['arn']
   external_id = data['external_id']
 
+
   credentials = get_ec2_credentials(arn, external_id)
-  upload_image_to_ec2(image_path, credentials, region, filename)
+  upload_image_to_ec2(image_path, credentials, filename, data)
```

Comment:
credentials are also fetched from data. So I would move this over too.

Reply:
No, credentials are calculated by get_ec2_credentials

Reply:
From the data variable. So it could be done in upload_image_to_ec2